string local_date
string local_time
string topic_msgs